openshift/openshift/src/main/java/io/undertow/protocols/spdy/SpdyChannelWithoutFlowControl.java

```diff
@@ -0,0 +1,45 @@
+package io.undertow.protocols.spdy;
```

Comment:
Yes, but I'm overriding a package accessible method. :(  This is only required because the OpenShift SPDY implementation does not support flow control, so once the window is exhausted, we stop forwarding data.  I wish there was an easier way (and maybe there is, but I haven't found it).